src/dao/pages/UpdateDAODetails/UpdateDAODetailsForm.tsx

```diff
@@ -83,6 +84,23 @@ export function UpdateDAODetailsForm() {
           fallback={<DefaultLogoIcon boxSize="4rem" alignSelf="end" color={Color.Grey_Blue._100} />}
         />
       </Flex>
+      {type !== DAOType.MultiSig && (
+        <FormInput<"infoUrl">
+          inputProps={{
+            id: "infoUrl",
+            label: "Info Url",
+            type: "text",
+            placeholder: "Enter the info url of your DAO",
+            register: {
+              ...register("infoUrl", {
+                required: { value: true, message: "A info url is required." },
+              }),
```

Comment:
we could add validation for Info URL i.e,. it should be a valid URL

Reply:
Done.